offlineimap.py

```diff
@@ -14,7 +14,7 @@
 #
 #    You should have received a copy of the GNU General Public License
 #    along with this program; if not, write to the Free Software
-#    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA
```

Comment:
Hmm, this has the same problem as the previous address, it could change at some point and it would require updating tons of source code again. Please use instead something like this:

     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <https://www.gnu.org/licenses/>.

@jelmer please update your scripts to this variant! :)

Reply:
Hey @guillemj, that's a good point. I'll update the script to point at the URL before I run it again.

Reply:
Not sure the URI is less prone to change than the postal address.

Reply:
Well, the GPL-3 already uses the URL instead of the postal address. And an URL has the advantage of supporting being redirected in case they decided to change it, which seems extremely unlikely given the amount of references out there, compared to physical relocation which might be a necessity. :)

I personally switched all projects I'm involved with to use the URL some time ago.